test/JDBC/expected/case_insensitive_collation-vu-prepare.out

```diff
@@ -0,0 +1,38 @@
+-- BABEL-3094
+-- sys table and view columns need to have the correct output type and collation
+create table t_rcv15 ([Test] varchar);
+go
+
+CREATE TABLE CaMeLtAbLe(cAmElCoLuMn VARCHAR(100) NOT NULL);
+go
+
+CREATE VIEW CaMeLvIeW AS SELECT 'VIEW' + cAmElCoLuMn + 'VIEW' AS CaMeLcOlUmNiNvIeW FROM CaMeLtAbLe;
+go
+
+-- BABEL-3165
+-- sys.databases.name should have case-insensitive collation
+create database TEST;
+go
+
+-- BABEL-3323
+-- When any sys view contains collatable constants
+create view vB as ( SELECT CASE WHEN 'b' = 'B' THEN 'case-insensitive' ELSE 'case-sensitive' END);
+go
```

Comment:
we need to add upgrade tests for this particular jira. Prepare this view in 13.x/ and then run the same verification on latest version.

Reply:
done